doc/rst/language/spec/ranges.rst

```diff
@@ -920,9 +883,10 @@ Range Slicing
 
 Ranges can be *sliced* using other ranges to create new sub-ranges. The
 resulting range represents the intersection between the two ranges’
-represented sequences. The stride and alignment of the resulting range
-are adjusted as needed to make this true. ``idxType`` and the sign of
-the stride of the result are determined by the first operand.
+represented sequences. The stride, alignment, and bounds of the
```

Comment:
I am not familiar with these semantics, would recommend someone else to review the changes to ranges. Generally, the wording looks good.